src/test/java/uk/gov/hmcts/sptribs/common/event/CreateTestCaseTest.java

```diff
@@ -30,6 +32,8 @@ class CreateTestCaseTest {
     @InjectMocks
     private CreateTestCase createTestCase;
 
+    @Mock
+    private CcdSupplementaryDataService coreCaseApiService;
```

Comment:
Supplementary data may not be required for case linking

Reply:
Removed this as its not required.